src/themes/schulportal/login/messages/messages_de.properties

```
@@ -29,4 +29,5 @@ passwordRequirementUpper=Mindestens 1 Großbuchstaben enthalten
 passwordRequirementDigit=Mindestens 1 Ziffer enthalten
 passwordRequirementHistory=Nicht dem bisherigen Passwort entsprechen
 passwordRequirementNoWhitespace=Keine Leerzeichen enthalten
-systemStatus=Systemstatus
\ No newline at end of file
+systemStatus=Systemstatus
+userNotFound=Diese Aktion setzt eine Zweifaktor-Authentifizierung (2FA) voraus. Bitte melden Sie sich im Schulportal an und richten dort eine 2FA ein oder wenden Sie sich an Ihren schulischen Administrator/Ihre schulische Administratorin.
```

```suggestion
userNotFound=Diese Aktion setzt eine Zweifaktor-Authentifizierung (2FA) voraus. Bitte melden Sie sich im Schulportal an und richten dort eine 2FA ein oder wenden Sie sich an Ihre schulischen Administratorinnen und Administratoren.
```
Damit das vereinheitlicht wird